targets/lenovo-x1/netvmExtraModules.nix

```diff
@@ -8,62 +8,75 @@
   microvm,
   hostConfiguration,
   ...
-}: {
-  imports = [./sshkeys.nix];
-  microvm = {
-    shares = [
-      {
-        tag = config.ghaf.security.sshKeys.waypipeSshPublicKeyName;
-        source = config.ghaf.security.sshKeys.waypipeSshPublicKeyDir;
-        mountPoint = config.ghaf.security.sshKeys.waypipeSshPublicKeyDir;
-      }
-    ];
+}: let
+  configH = config;
```

Comment:
Either repleace this line with `configH = hostConfiguration.config` or just remove the `configH` altogether and replace later occurences of `configH` with `hostConfiguration.config`